boards/arm/rtl8762gn_evb/rtl8762gn_evb.dts

```diff
@@ -0,0 +1,55 @@
+/dts-v1/;
+#include <rtk/rtl8762gn_evb.dtsi>
```

Comment:
Please change the dts/arm/rtk/rtl8762gn_evb.dtsi to dts/arm/**realtek**/rtl8762gn_evb.dtsi

Reply:
Done